src/main/kotlin/com/sourcegraph/cody/listeners/CodyCaretListener.kt

```diff
@@ -14,7 +15,7 @@ import com.sourcegraph.utils.CodyEditorUtil
 
 class CodyCaretListener(val project: Project) : CaretListener {
   override fun caretPositionChanged(e: CaretEvent) {
-    if (!ConfigUtil.isCodyEnabled()) {
+    if (!ConfigUtil.isCodyEnabled() || e.editor.editorKind != EditorKind.MAIN_EDITOR) {
```

Comment:
The other are:

```
public enum EditorKind {
  UNTYPED,
  MAIN_EDITOR,  // instead of SoftWrapAppliancePlaces.MAIN_EDITOR
  CONSOLE,      // EDITOR_IS_CONSOLE_VIEW, SoftWrapAppliancePlaces.CONSOLE
  PREVIEW,      // SoftWrapAppliancePlaces.PREVIEW
  DIFF         // EDITOR_IS_DIFF_KEY
}
```